regression/scripts/Vehicles/Cessna_172.py

```diff
@@ -168,95 +188,100 @@ def vehicle_setup():
+    main_gear                                   = SUAVE.Components.Landing_Gear.Main_Landing_Gear()
+    nose_gear                                   = SUAVE.Components.Landing_Gear.Nose_Landing_Gear()
+    main_gear.strut_length                      = 12. * Units.inches #guess based on picture
+    nose_gear.strut_length                      = 6. * Units.inches 
+                                                
+    landing_gear.main                           = main_gear
+    landing_gear.nose                           = nose_gear
+                                                
+    #add to vehicle                             
+    vehicle.landing_gear                        = landing_gear
+                                                
+    # ------------------------------------------------------------------
+    #   Fuel
+    # ------------------------------------------------------------------    
+    # define fuel weight needed to size fuel system
+    fuel                                        = SUAVE.Attributes.Propellants.Aviation_Gasoline()
+    fuel.mass_properties                        = SUAVE.Components.Mass_Properties() 
+    fuel.number_of_tanks                        = 1.
+    fuel.origin                                 = wing.origin
+    fuel.internal_volume                        = fuel.mass_properties.mass/fuel.density #all of the fuel volume is internal
+    fuel.mass_properties.center_of_gravity      = wing.mass_properties.center_of_gravity
+    fuel.mass_properties.mass                   = 319 *Units.lbs
+    vehicle.fuel                                = fuel
 
-    #add to vehicle
-    vehicle.landing_gear   = landing_gear
+    # ------------------------------------------------------------------
+    #   Piston Propeller Network
+    # ------------------------------------------------------------------    
+    
+    # build network
+    net                                         = SUAVE.Components.Energy.Networks.Internal_Combustion_Propeller()
+    net.tag                                     = 'internal_combustion'
+    net.number_of_engines                       = 1.
+    net.nacelle_diameter                        = 42 * Units.inches
+    net.engine_length                           = 0.01 * Units.inches
+    net.areas                                   = Data()
+    net.rated_speed                             = 2700. * Units.rpm
+    net.rated_power                             = 180.  * Units.hp
+    net.areas.wetted                            = 0.01
+                                                
+    # Component 1 the engine                    
+    net.engine                                  = SUAVE.Components.Energy.Converters.Internal_Combustion_Engine()
+    net.engine.sea_level_power                  = 180. * Units.horsepower
+    net.engine.flat_rate_altitude               = 0.0
+    net.engine.speed                            = 2700. * Units.rpm
+    net.engine.BSFC                             = 0.52 
```

Comment:
This should be written out in the appropriate file.

Reply:
"BSFC" changed to "power_specific_fuel_consumption" . I had changed the Energy component but forgot to change the vehicle file

Reply:
In this case how did the regression still run properly?

Reply:
It still ran because despite not being initialized in the default settings, it was defined by the user and the function still used the argument "BSFC". Issue corrected